src/components/Analysis/ContigViewer/Filter/ContigLength/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useEffect, useState, useContext, memo } from 'react';
+
+import Slider from 'components/UI/Slider';
+import Switch from 'components/UI/Switch';
+import ContigsQueryContext from 'components/Analysis/ContigViewer/ContigsQueryContext';
+
+const MIN = 500;
+const MAX = 100000;
```

Comment:
Any chance the length of the contigs is bigger than this?

Reply:
Turns out, yes! Just pushed support for lengths up to 10E6 (I believe that is longer than what we have). This needed a logarithmic scale on the slider to be usable over the range of expected contig lengths.